### ISSUE TITLE
Missing regularization in flowers model

### ISSUE THREAD

**Issue by EffePen** [GoogleCloudPlatform/cloudml-samples#34]
This is not exactly an issue, more a missing feature: In the flowers model, the loss function only takes into account data loss and no regularization (such as L2 regularization) is possible except dropout. Currently I am retraining the model with my own dataset and it suffers from overfitting, therefore the addition of regularization might help.
Is this choice due to ML considerations or the regularization has not been included just to keep the example simple and because the flowers-dataset didn't need it?

**Comment by joshgc** [comment]
Good point.  We didn't introduce regularization for simplicity (it does OK on 3000 images in flowers/) and to keep consistency with the original [Tensorflow code retrain.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/image_retraining/retrain.py).

I'd love to hear about which regularization you chose and how it performed.  Could you run it against flowers too?

**Comment by puneith** [comment]
Closing this issue because of inactivity and that its not really an issue.
